packages/twenty-server/src/engine/core-modules/twenty-config/config-variables.ts

```diff
@@ -979,11 +979,21 @@ export class ConfigVariables {
   @ConfigVariablesMetadata({
     group: ConfigVariablesGroup.LLM,
     isSensitive: true,
+    isEnvOnly: true,
     description: 'API key for OpenAI integration',
     type: ConfigVariableType.STRING,
   })
   OPENAI_API_KEY: string;
 
+  @ConfigVariablesMetadata({
+    group: ConfigVariablesGroup.LLM,
+    isSensitive: true,
+    isEnvOnly: true,
```

Comment:
Set `isEnvOnly` to false, that's what will allow people to configure it in the admin panel